examples/digital_fingerprinting/production/morpheus/dfp/modules/dfp_pipeline_preprocessing.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import dfp.modules.dfp_preprocessing  # noqa: F401
+import dfp.modules.dfp_rolling_window  # noqa: F401
+import dfp.modules.dfp_split_users  # noqa: F401
+import dfp.modules.dfp_training  # noqa: F401
```

Comment:
Why do we need noqa here?

Reply:
Without noqa, CI fails because it seems as though the imports aren't being used but the modules are being registered.

Reply:
Without noqa CI is failing because of unused imports but they actually using to register the module.